overview/saml.mdx

```diff
@@ -15,6 +15,45 @@ which users have access to your product. They could give everyone at the company
 access, specify teams like only the engineering team, or even pick specific
 people to give access.
 
+### SAML Setup Guide
+
+
+
+1. In the Organization Settings for your project, make sure that SAML is enabled and that all orgs can setup SAML.
```

Comment:
There's actually a choice here. You can either set it up so that all orgs can setup SAML or you can enable SAML for each individual org. Doing it this way is fine, but I'd just add a line saying that you can also enable SAML on a per-org basis by enabling `Can setup SAML` in the dashboard.